packages/gatsby-telemetry/src/__tests__/error-helpers.js

```diff
@@ -0,0 +1,90 @@
+const { sanitizeError, cleanPaths } = require(`../error-helpers`)
+
+describe(`Errors Helpers`, () => {
+  describe(`sanitizeError`, () => {
+    it(`Removes env, output and converts buffers to strings from execa output`, () => {
+      const tags = {
+        error: [
+          {
+            error: null,
+            cmd: `git commit -m"test"`,
+            file: `/bin/sh`,
+            args: [`/bin/sh`, `-c`, `git commit -m"test`],
+            options: {
+              cwd: `here`,
+              shell: true,
+              envPairs: [`VERSION=1.2.3`],
+              stdio: [{}, {}, {}], // pipes
+            },
+            envPairs: [`VERSION=1.2.3`],
+
+            stderr: Buffer.from(`this is a test`),
```

Comment:
choose another message for the error so we are sure we don't do anything stupid in our sanitize errors
```suggestion
            stderr: Buffer.from(`this is a an error`),
```